templates/project/Cargo.toml

```diff
@@ -18,23 +18,11 @@ opt-level = "z"
 embedded-svc = "0.27"
 embedded-hal = { version = "0.2.7", features = ["unproven"]}
 log = "0.4"
-async-channel = "1.8.0"
-futures-lite = "1.12.0"
-micro-rdk = {version = "0.1.8", git = "https://github.com/viamrobotics/micro-rdk.git", features = ["esp32", "binstart"], rev = "c669b06" }
+async-channel = "2"
+futures-lite = "1"
+micro-rdk = {version = "0.1.8", git = "https://github.com/viamrobotics/micro-rdk.git", features = ["esp32", "binstart","provisioning"], rev = "e5ba39f" }
 
 [build-dependencies]
 anyhow = "1"
-cargo_metadata = "0.18.1"
-const-gen = "1.3.0"
+cargo_metadata = "0.18"
```

Comment:
⬆️ - Do you want to drop the `anyhow` dependency here too?

Reply:
done